mediainfojs.cpp

```diff
@@ -18,15 +18,29 @@ class MediaInfoJs {
   int open_buffer_continue(const std::string& data, double size) {
     return mi.Open_Buffer_Continue((ZenLib::int8u*)data.data(), (ZenLib::int64u)size);
   }
-  long open_buffer_continue_goto_get(){
-    return (int64_t)mi.Open_Buffer_Continue_GoTo_Get();
+  int open_buffer_continue_goto_get(){
+    seekTo.total = mi.Open_Buffer_Continue_GoTo_Get();
+    return seekTo.lower;
+  }
+  int get_seek_to_upper() {
```

Comment:
I recommend `open_buffer_continue_goto_get_high` in order to better see the relation with the low version.
The low version could be called `open_buffer_continue_goto_get_low` as it breaks the expect behavior (a 64-bit integer) from upstream `open_buffer_continue_goto_get`
`open_buffer_continue_goto_get` code could be kept for legacy reasons.

Reply:
@JeromeMartinez Would `open_buffer_continue_goto_get_high` also invoke `Open_Buffer_Continue_GoTo_Get`? The name infers that it should but I'm not sure if that's correct or what the implication of that behaviour is.

Reply:
Good catch.
Now I recommend, if it is possible on the glue side (I did not test), to remove the struct and call C `Open_Buffer_Continue_GoTo_Get()` call for both low and high request, it would avoid the intermediate value which may be dangerous (depending on which order is used for getting high or low value).

`Open_Buffer_Continue_GoTo_Get()` can be called several times, it returns (very quickly) the same value until a parsing call is made.